fr.free.Qccrypt.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Philippe Beaureilles -->
+<component type="desktop-application">
+  <id>fr.free.Qccrypt</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
+  <name>Qccrypt</name>
+  <summary>QT based graphical frontend for ccrypt tool</summary>
+  <developer_name>Philippe Beaureilles</developer_name>
+  <description>
+    <p>QT based graphical frontend for ccrypt tool.</p>
+  </description>
+  <launchable type="desktop-id">fr.free.Qccrypt</launchable>
```

Comment:
this is an invalid launchable. it should be the file name of the .desktop file.

Reply:
Thank you for your help, I will change it. I'm running out of time right now, so I will make all changes you mentioned in the next few days. Thanks again for your great help!